server/app/routes/api/v1/users/index.js

```diff
@@ -1,27 +1,55 @@
 'use strict';
-var router = require('express').Router();
-module.exports = router;
-var _ = require('lodash');
+var router = require( 'express' )
+  .Router();
+var _ = require( 'lodash' );
+
+const sequelizeHandlers = require( 'sequelize-handlers' );
+const bluebird = require( 'bluebird' );
+const utils = require( '../../../utils' );
+const {
+  ensureAuthenticated,
+  ensureIsAdmin,
+  Credentials,
+  respondWith404,
+  _err,
+  db
+} = utils;
+const models = db()
+  .models;
+const {
+  user: User,
+  team: Team,
+  organization: Organization
+} = models;
+const Resource = User;
+
+router.use( ( req, res, next ) => {
+  req.options = {
+    include: [
+      Team, Organization
+    ],
+    attributes: {
+      exclude: [
+        'email', 'password', 'salt'
+      ]
+    }
+  }
+  next();
+} )
 
-router.get('/', (req, res, next)=> res.send("happy day!"))
+/** see documentation at https://www.npmjs.com/package/sequelize-handlers */
+router.get( '/', sequelizeHandlers.query( Resource ) );
+router.get( '/:id', ( req, res, next ) => {
+  if ( req.user && req.user.id === req.params.id ) {
+    req.options.attributes.exclude = [ 'salt' ];
```

Comment:
Very nice, then you might want to consider a utility function 'isSelf(id)' that takes an id and determines if that user is the req.user. I feel like you might use that in more places (if not, then no need for a utility function)